src/node.h

```diff
@@ -306,9 +306,17 @@ class NODE_EXTERN InitializationResult {
 // better suited for a public embedder API.
 NODE_EXTERN int Start(int argc, char* argv[]);
 
+enum StopFlags {
```

Comment:
This should probably be an `enum class` or put in a namespace like `ProcessInitializationFlags`, so that the flag names don’t get hoisted to the `node` namespace

Reply:
@addaleax in that case, should i rename `ProcessInitializationFlags` back to `ProcessFlags`?

Reply:
Ah no, the idea was not to share the same namespace but to create a similar one for this enum, i.e.
```c++
namespace StopFlags {
enum Flags {
...
};
}
```

so that the individual flags could then be referred to as `StopFlags::kDoNotTerminateIsolate` etc.

Reply:
ah gotcha! ok, should be set now.